examples/validation-app/README.md

```diff
@@ -0,0 +1,49 @@
+# @loopback/example-validation-app
+
+An example application to demonstrate validation in LoopBack.
+
+## Summary
+
+This application shows how to add validation in a LoopBack application. It
+exposes `/coffee-shops` endpoints to create/read/update/delete a CoffeeShop
+instance with the in-memory storage.
+
+### Key artifacts
+
+- [CoffeeShop model](src/models/coffee-shop.model.ts)
+  - Shows how to add validation using [AJV](https://www.npmjs.com/package/ajv)
+- [ValidatePhoneNumInterceptor](src/interceptors/validate-phone-num.interceptor.ts)
+  and [CoffeeShopController](src/controllers/coffee-shop.controller.ts)
+  - Shows how to add reusable validation logic in interceptors
+
+## Use
+
+Start the app:
+
+```sh
+npm start
+```
+
```

Comment:
nitpick: maybe add more tips about the expected format of each field?

Like:
- phone number: xxx-xxx-xxxx, otherwise gets 422
and same for city and area code.

Reply:
will add. per Dom's suggestion as well.